engine/src/main/java/org/terasology/config/BindsConfig.java

```diff
@@ -54,7 +54,7 @@
  * One example for a binding-input combination is the {@link ForwardsButton} 
  * which is defined in the engine with the id <code>forwards</code>.
  * The default input binding for the forward movement is the W-Key.
- * Therefore the binds for <code>engine:forwards</code> would contain an {@link Input} object with type {@link InputType#KEY} and id {@link KeyId.W}.
+ * Therefore the binds for <code>engine:forwards</code> would contain an {@link Input} object with type {@link InputType#KEY} and id {@link KeyId}.
```

Comment:
"W" is a valid member of KeyId, and is correctly mentioned; it should be changed to KeyId#W, which is the correct JavaDoc syntax, rather than be removed.

Reply:
Ahh, yes. Thanks for that. Now trying to figure out how to add commits to an existing PR.